sass/tokens/register/_math.scss

```diff
@@ -4,12 +4,15 @@
 
 // Plus
 // ----
-/// Add two values together in accoutrement maps.
+/// Add two values together in Accoutrement maps.
 ///
-/// Direct access to this function is prefixed,
-/// but it is available in all accoutrement maps as
+/// Available in all Accoutrement maps as
 /// `'plus'`, `'add'`, or `'+'`.
 ///
+/// @since 4.0.0 -
+/// - BREAKING: Renamed from `_a_plus` to `plus`
```

Comment:
When we move something from private to public, we don't have to document it as a breaking change - just a new feature.